vncviewer/parameters.cxx

```diff
@@ -164,7 +164,7 @@ BoolParameter fullscreenSystemKeys("FullscreenSystemKeys",
                                    true);
 
 #ifndef WIN32
-StringParameter via("via", "Gateway to tunnel via", "");
+StringParameter via("via", "SSH host for tunnel", "");
```

Comment:
This is an unrelated change, so it should be a separate commit.

The term "Gateway" is also used elsewhere, so everything would need to be adjusted so things are consistent.

Lastly, I'm uncertain if this is clearer. We're gaining the detail of it being SSH, but losing the gateway¹ aspect.

¹ Which is a less common description than "proxy"

Reply:
I'll remove this particular change.